### PR TITLE
TASK-58038: Add based swagger ui rest documentation

### PR DESCRIPTION
**Pull request by hakermi**
Add based swagger ui rest documentation

### CODE REVIEW

File: docs/Swagger_Rest_Apis.rst [hunk 1]
```diff
@@ -12,12 +12,51 @@ eXo Rest APIs
...
+
+ :ref:`Tasks Addon Rest API <TASKS_ADDON_REST_API>`
+
+ :ref:`Wallet Addon Rest API <WALLET_ADDON_REST_API>`
+
+ :ref:`Gamification Addon Rest API <GAMIFICATION_ADDON_REST_API>`
+
+ :ref:`Processes Addon Rest API <PROCESSES_ADDON_REST_API>`
+
+ :ref:`Documents Addon Rest API <PROCESSES_ADDON_REST_API>`
```
Comment (ahamdi) [inline docs/Swagger_Rest_Apis.rst]: this should Documents Rest API

File: docs/Swagger_Rest_Apis.rst [hunk 2]
```diff
@@ -12,12 +12,51 @@ eXo Rest APIs
 
  :ref:`Commons Rest API <COMMONS_REST_API>`
 
+ :ref:`Ecms Rest API <ECMS_REST_API>`
```
Comment (ahamdi) [inline docs/Swagger_Rest_Apis.rst]: Could you sort them by first letter

Comment (hakermi) [reply docs/Swagger_Rest_Apis.rst]: done

File: docs/Swagger_Rest_Apis.rst (hunk 1 above)
Comment (hakermi) [reply docs/Swagger_Rest_Apis.rst]: done

**Review by ahamdi — APPROVED** [review]

**Pull request merged**